pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,58 +1,396 @@
 <section class="my-4">
+  {{ if .Envelope }}
   <section>
     <div class="mb-4 md:mb-0">
-      <span class="font-semibold">Transaction ID: {{ .ID }}</span>
+      <span class="font-semibold">Transaction ID: {{ .Envelope.Transaction.Txid }}</span>
```

Comment:
The `Envelope` struct may be found [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/api/v1/transaction.go#L69-L76).